test/cluster/mzcompose.py

```diff
@@ -169,13 +170,25 @@ def test_github_12251(c: Composition) -> None:
 
 
 def test_remote_storaged(c: Composition, redpanda: bool) -> None:
-    with c.override(Testdrive()):
-        dependencies = ["materialized", "storaged"]
+    with c.override(
+        Testdrive(default_timeout="15s", no_reset=True, consistent_seed=True),
+        # Use a separate PostgreSQL service for persist rather than the one in
+        # the `Materialized` service, so that crashing `environmentd` does not
+        # also take down PostgreSQL.
+        Postgres(),
+        Materialized(
```

Comment:
@philip-stoev Reminds me, should we rename all of these to `Environmentd` by now? Also below, we still have, for example, `c.up("materialized")`.

Reply:
Ah yeah, it's a bit complicated. We should add a _new_ service called `Environmentd` that doesn't attempt to start a Postgres container (i.e., uses the `environmentd` image instead of the `materialized` image). But `Materialized` is meant to live on as the name of the all-in-one Docker image.